tests/components/geo_json_events/__init__.py

```diff
@@ -1 +1,15 @@
 """Tests for the geo_json_events component."""
+from unittest.mock import MagicMock
+
+
+def _generate_mock_feed_entry(
+    external_id, title, distance_to_home, coordinates, properties=None
+):
```

Comment:
It would be better with type hints:
```suggestion
    external_id: xxx, title: xxx, distance_to_home: xxx, coordinates: xxx, properties: xxx | None=None
) -> None:
```

Reply:
OK, I'll add type hints.